src/Microsoft.TestPlatform.ObjectModel/RunSettings/RunConfiguration.cs

```diff
@@ -600,7 +598,7 @@ public static RunConfiguration FromXml(XmlReader reader)
                         XmlRunSettingsUtilities.ThrowOnHasAttributes(reader);
                         string collectSourceInformationStr = reader.ReadElementContentAsString();
 
-                        bool bCollectSourceInformation = true;
+                        bool bCollectSourceInformation;
```

Comment:
if parse fail should this be `true`?

Reply:
If parse fails we throw an exception (this is only a local variable not a field).